tensorflow/contrib/gdr/gdr_memory_manager.cc

```diff
@@ -276,8 +275,13 @@ Status GdrMemoryManager::Init() {
     };
     GPUProcessState::singleton()->AddGPUAllocVisitor(bus_id,
                                                      cuda_alloc_visitor);
-    GPUProcessState::singleton()->AddCUDAHostAllocVisitor(bus_id,
-                                                          alloc_visitor);
+
+    for (int numa_idx = 0; numa_idx < port::NUMANumNodes(); ++numa_idx) {
+      GPUProcessState::singleton()->AddCUDAHostAllocVisitor(numa_idx,
+                                                            alloc_visitor);
+      LOG(INFO) << "Instrumenting Cuda host allocator on numa " << numa_idx;
+    }
+
     GPUProcessState::singleton()->AddCUDAHostFreeVisitor(bus_id, free_visitor);
```

Comment:
Should rename `bus_id` to `numa_idx`, and move it inside the loop.

Reply:
@poxvoculi I saw different signatures for `AddGPUAllocVisitor(int bus_id, const SubAllocator::Visitor& visitor)` and `AddCUDAHostAllocVisitor(int numa_node, const SubAllocator::Visitor& visitor)`. It seems rather confusing.

Maybe change `bus_id` to `numa_node` for `AddGPUAllocVisitor` as well?